mvn_setup/tasks-parent/tasks-ui/src/core/TaskTable/CellMenu.tsx

```diff
@@ -23,23 +34,22 @@ const HoverMenu: React.FC<{ onEdit: () => void }> = ({ onEdit }) => {
               <FormattedMessage id={`tasktable.menu.edit`} />
             </ListItemText>
           </MenuItem>
-          <MenuItem>
-            <ListItemText>OPTION2</ListItemText>
-          </MenuItem>
-          <MenuItem>
-            <ListItemText>OPTION With longer text 3</ListItemText>
-          </MenuItem>
-
-          <Divider />
-          <MenuItem>
-            <ListItemText>OPTION4</ListItemText>
+          <MenuItem onClick={onWork}>
```

Comment:
This `<MenuItem><ListItemText><FormattedMessage>` pattern seems to be repeating.  So let's make it a bit less so.

These things should be extracted to a new React component.  

Then, you can call them inside of HoverMenu something like this:

```
const HoverMenu: React.FC<{
  onEdit: () => void,
  onWork: () => void,
}> = ({ onEdit, onWork }) => {
  const Popover = usePopover();

  return (
    <>
      <Popover.Delegate>
        <MenuList dense>
          <MyMenuItem title={'tasktable.menu.xxx'} onClick={...} />
          <MyMenuItem title={'tasktable.menu.xxx'} onClick={...} />
          <Divider />
          ...
```